package/MDAnalysis/core/selection.py

```diff
@@ -567,6 +567,29 @@ def _apply(self, group):
         return group[mask]
 
 
+class SingleCharSelection(Selection):
+    """for when an attribute is just a single character, eg RSChirality
+
+    .. versionadded:: 2.1.0
+    """
+    def __init__(self, parser, tokens):
+        super().__init__(parser, tokens)
+        vals = grab_not_keywords(tokens)
+        if not vals:
+            raise ValueError("Unexpected token '{0}'".format(tokens[0]))
+
+        self.values = vals
+
+    @return_empty_on_apply
+    def _apply(self, group):
+        attr = getattr(group, self.field)
+
+        mask = (attr == self.values[0])
+        for v in self.values[1:]:
+            mask |= (attr == v)
```

Comment:
Any reason not to use np.isin ?
```suggestion
        mask = np.isin(attr, self.values)
```

Reply:
Oh wait I'm not sure if self.values will be a string like `'R S'` or a list like `['R', 'S']`. It needs to be a list to work with np.isin

Reply:
`self.values` will be a list, as we consume all tokens until the next keyword.